editor/import/3d/resource_importer_scene.cpp

```diff
@@ -771,8 +773,53 @@ Node *ResourceImporterScene::_pre_fix_node(Node *p_node, Node *p_root, HashMap<R
 	if (!use_node_type_suffixes) {
 		return p_node;
 	}
+	if (_teststr(name, "occ") || _teststr(name, "occonly")) {
```

Comment:
I re-ordered the node type suffixes in `ResourceImporterScene::_pre_fix_node` to have the physics ones grouped together at the bottom. Previously the order was Physics -> Navigation -> Occluder -> Physics Again. Now the order is Occluder -> Navigation -> Physics. This doesn't change behavior as these suffixes all have unique endings.